scripts/src/create_data/offers.ts

```diff
@@ -118,6 +120,10 @@ export async function createEarn(
 		console.log("Updating earn offer %s id %s", offer.name, offer.id, options.dryRun ? "(dry run)" : "");
 		content = await OfferContent.findOne({ offerId: offer.id });
 	} else {
+		if (options.onlyUpdate) {
+			console.log(`Skipping offer creation for offer: ${offerName}`);
```

Comment:
what's going on???  didn't we say for like a million times that we have spaces between curly brackets???